descriptors/ModuleDescriptor-template.json

```diff
@@ -222,26 +222,10 @@
       "id" : "login",
       "version" : "6.0 7.0"
     },
-    {
-      "id" : "service-points",
-      "version" : "2.1 3.0"
-    },
-    {
-      "id" : "service-points-users",
-      "version" : "1.0"
-    },
-    {
-      "id": "password-validator",
-      "version": "1.0"
-    },
     {
       "id": "authtoken",
       "version": "2.0"
     },
-    {
-      "id": "notify",
-      "version": "2.0"
-    },
```

Comment:
What is the intended behaviour of the forgotten password process when there is no ability to notify the user?

Do we consider the system to be working as intended when there is no way for a user to recover from a forgotten password using this process because they are never notified?

Reply:
The actual intention is that we will have both these interfaces in most functional FOLIO setups, but neither of them when using FOLIO in development as a context for working on other modules. I don't envisage any situation where you would have `password-validator` but not `notify`.

Reply:
> The actual intention is that we will have both these interfaces in most functional FOLIO setups, but neither of them when using FOLIO in development as a context for working on other modules. I don't envisage any situation where you would have `password-validator` but not `notify`.

I don't understand how that is relevant. Development isn't the only use case for a minimal platform.

The system defined by that minimal platform should _still be viable and work_  under any valid configuration.

Given that mod-users-bl provides the ability to reset a users password and that can _only_ work when both of those interfaces are present, those interfaces are required dependencies for the module to achieve it's basic function.